### ISSUE TITLE
Icons for GPS, cell tower, wifi and observations no longer make sense to display

### ISSUE THREAD

**Issue by crankycoder** [mozilla/MozStumbler#1766]
We originally created these icons at the top of the stumbler to give users a way to aid them in gaining points on the old leaderboard.

The new leaderboard only cares about total number of reports, we should probably remove the icons, or better yet - simplify them to be more meaningful.

**Comment by cascheberg** [comment]
I think those are interesting data, even if they are not relevant for the leaderboard.

**Comment by cascheberg** [comment]
How are points actually calculated in the new leaderboard?
In the latest version of the old leaderboard the gained points exactly matched the number of new observations (i.e. locations) as shown in the app.
But in the current new leaderboard I gain much more points than just the observation count.

---

At one point the location-leaderboard readme states: "Submit the number of networks detected within a series of tiles for a given contributor."
The app seems to simply increment the reported `observations` by one for each `ACTION_NEW_BUNDLE` intent.

**Comment by crankycoder** [comment]
cc/ @jaredkerim

**Comment by breversa** [comment]
Agreed with @cascheberg : those icons are useful, even when not competing for leaderboard, simply to see at a glance how many APNs, cell towers or reports your device sees/made.
